string.c

```diff
@@ -11341,17 +11354,29 @@ enc_str_scrub(rb_encoding *enc, VALUE str, VALUE repl, int cr)
 
 /*
  *  call-seq:
- *    str.scrub -> new_str
- *    str.scrub(repl) -> new_str
- *    str.scrub{|bytes|} -> new_str
+ *    scrub(replacement_string = default_replacement) -> string or self
+ *    scrub{|bytes| ... } -> string or self
```

Comment:
`scrub` never returns self, it always returns a potentially modified copy of the receiver.

Reply:
Done.